changelog.d/1081.change.rst

```diff
@@ -0,0 +1 @@
+Fix frozen exception classes when raised within e.g. `contextlib.contextmanager`, which mutates their ``__traceback__`` attributes.
```

Comment:
Sir this is a ~Wendy's~ Markdown shop. 🤓

Reply:
Ups, fix fixed then.